ui/.env.development

```diff
@@ -1,2 +1,4 @@
-REACT_APP_MIXPANEL_TOKEN=6b8d3fb93ffd04f2e76325540c838624
-PORT=3001
+REACT_APP_VERSION=$npm_package_version
+REACT_APP_NAME=$npm_package_name
+REACT_APP_SENTRY_DSN=https://10930d3b0a8d427cad2147d6d845be56@o226447.ingest.sentry.io/5721090
+REACT_APP_SENTRY_ENABLED=false
```

Comment:
Not big deal, but better to leave a more blank line.

Reply:
fixed, vscode was auto removing that new line for no reason..